src/src.pro

```diff
@@ -6,43 +6,42 @@ CONFIG += qt thread silent
 CONFIG += c++11
 DEFINES += BOOST_NO_CXX11_RVALUE_REFERENCES
 
-# Windows specific configuration
+# Platform specific configuration
 win32: include(../winconf.pri)
-
-# Mac specific configuration
 macx: include(../macxconf.pri)
-
-# Unix specific configuration
 unix:!macx: include(../unixconf.pri)
-
-# eCS(OS/2) specific configuration
 os2: include(../os2conf.pri)
 
+QT += network xml
+
 nogui {
+    TARGET = qbittorrent-nox
     QT -= gui
     DEFINES += DISABLE_GUI
-    TARGET = qbittorrent-nox
 } else {
-    macx: QT += macextras
-    macx: LIBS += -lobjc
+    TARGET = qbittorrent
     QT += xml concurrent svg widgets
+
     CONFIG(static) {
         DEFINES += QBT_STATIC_QT
         QTPLUGIN += qico
     }
     win32 {
         QT += winextras
     }
-    TARGET = qbittorrent
+    macx {
```

Comment:
Maybe move this to macxconf.pri?

Reply:
I thought about this but not sure with this move, placing `QT` defines together at here is not that bad ...

@sledgehammer999 
any comment?

Reply:
Okay, no need to linger over this... We can do it later, if needed.